text/0008-sync-and-storage-review-packet.md

```diff
@@ -12,35 +17,20 @@ Is this roadmap proposal congruent with Firefox's strategy?
 
 ## Document status
 
-*   2017-11-20: Go-ahead given.
-*   2017-11-09: Review successfully completed. Follow-up questions underway.
-*   2017-11-03: Sufficient contributions received; sent to the Reviewer.
+* 2017-11-20: Go-ahead given.
+* 2017-11-09: Review successfully completed. Follow-up questions underway.
+* 2017-11-03: Sufficient contributions received; sent to the Reviewer.
 
 ## Roles
 
-<dl>
-
-<dt>Reviewer</dt>
-
-<dd>dcamp</dd>
-
-<dt>Chair</dt>
-
-<dd>jwalker</dd>
-
-<dt>Proposers</dt>
-
-<dd>
-
-*   rnewman, Browser Architecture
-*   rfkelly, Engineering Lead, Firefox Accounts
-*   markh, Sync engineering
-*   emily, Browser Architecture
-*   adavis, Product Manager, Sync & FxA
-
-</dd>
-
-</dl>
+* **Reviewer**: dcamp
+* **Chair**: jwalker
+* **Proposers**: 
```

Comment:
The "official" term in the Architecture Review Process is "the team". I'm wondering if actually "the proposers" isn't a better phrase. @ncalexan @mykmelez do you have opinions?

Reply:
I like "the proposers".  Sounds like a great band, I'd pay 10 quid to see their show.